src/quantifiers.ts

```diff
@@ -36,9 +37,32 @@ export function zeroOrMore(...children: RegexComponent[]): ZeroOrMore {
   };
 }
 
+export function repeat(
+  config: Repeat['config'],
+  ...children: RegexComponent[]
+): Repeat {
+  return {
+    type: 'repeat',
+    children,
+    config,
```

Comment:
[nit] consider putting `count`, `min`, `max` directly here. Perhaps as a union type in some way.

Reply:
I'm not sure how is that supposed to work?  I think that `Repeat` type needs to have it